packages/relative-values/src/components/View/RelativeCell/DistCell.tsx

```diff
@@ -71,24 +78,30 @@ export const DistCell: FC<{
         >
           <div className="text-center z-0 py-1">
             <div>
-              <span className="text-slate-700 text-lg font-semibold">
-                <NumberShower number={item.median} precision={1} />
-              </span>
+              {showMedian && (
+                <span className="text-slate-700 text-lg font-semibold">
+                  <NumberShower number={item.median} precision={1} />
+                </span>
+              )}
               <span>
-                {" "}
-                <span
-                  style={{ fontSize: "0.7em" }}
-                  className="text-gray-400 font-light"
-                >
-                  ±
-                </span>{" "}
+                {showMedian && (
+                  <span>
+                    {" "}
+                    <span
+                      style={{ fontSize: "0.7em" }}
+                      className="text-gray-400 font-light"
+                    >
+                      ±
+                    </span>{" "}
+                  </span>
+                )}
                 <span
                   className={numberToColor(
                     item.uncertainty,
                     uncertaintyPercentiles
                   )}
                 >
-                  <NumberShower number={item.uncertainty} precision={2} />
+                  <NumberShower number={item.uncertainty / 2} precision={2} />
```

Comment:
I'm not sure how this happened, but it either should be moved to squiggle code, or deserves a comment.